lib/elixir/lib/exception.ex

```diff
@@ -177,6 +177,19 @@ defmodule Exception do
     end
   end
 
+  @doc false
+  @spec _format_message_with_term(String.t(), any) :: String.t()
+  def _format_message_with_term(message, term) do
+    inspected =
+      term
+      |> inspect(pretty: true)
+      |> String.split("\n", trim: true)
+      |> Enum.map(&("    " <> &1))
+      |> Enum.join("\n")
```

Comment:
I don't think we want to trim, because that will remove empty lines and change what was formatted. You either want to indent all lines, which means a `|> String.replace("\n", "\n      ")` or do:

```suggestion
      |> String.split("\n")
      |> Enum.map(fn
        "" -> ""
        line -> "    " <> line
      end)
      |> Enum.join("\n")
```